migrations/versions/f5c28956049b_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: f5c28956049b
+Revises: 43e8c59337ae
+Create Date: 2019-06-24 02:46:54.012871
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'f5c28956049b'
+down_revision = '43e8c59337ae'
```

Comment:
Change the down_revision to the latest migration

Reply:
@mrsaicharan1 done!